source/library/buttercup.js

```diff
@@ -6,7 +6,8 @@ import {
     Credentials,
     entryFacade
 } from "./buttercupCore.js";
-import AsyncStorageInterface from "../compat/AsyncStorageInterface.js";
+// import AsyncStorageInterface from "../compat/AsyncStorageInterface.js";
```

Comment:
I guess this can also be removed?

Reply:
Removed in 908fdcf1e8aa10f5c716fe73d028dbc6ed26d5e9